biz.aQute.bndlib/src/aQute/bnd/osgi/Domain.java

```diff
@@ -448,6 +450,36 @@ public Parameters getProvideCapability() {
 		return getParameters(Constants.PROVIDE_CAPABILITY, true);
 	}
 
+	public static Domain domain(byte[] data) {
+		try (JarInputStream jin = new JarInputStream(new ByteArrayInputStream(data))) {
```

Comment:
Some time past, we replaced all usage of ByteArrayInputStream with ByteBufferInputStream to avoid the unnecessary synchronization. Can you use ByteBufferInputStream here instead?